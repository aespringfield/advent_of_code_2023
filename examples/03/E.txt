............470.128...288...+......442-.......191.....%....&........360....218...............916.....552.171..747.51......111...............
