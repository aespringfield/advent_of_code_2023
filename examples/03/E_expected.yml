part_one: 442
